sklearn/inspection/_permutation_importance.py

```diff
@@ -78,9 +78,9 @@ def permutation_importance(estimator, X, y, scoring=None, n_repeats=5,
         `-1` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
-    random_state : int, RandomState instance, or None, default=None
-        Pseudo-random number generator to control the permutations of each
-        feature. See :term:`random_state`.
+    random_state : int, RandomState instance or None (default)
+        Pseudo-random number generator to control the permutations of each feature.
```

Comment:
please add

> pass an int to get reproducible results across function calls

Reply:
Oj, we thought about that and tried to understand but weren't sure!
Thanks!